Unity.gitignore

```diff
@@ -24,6 +24,7 @@
 
 # Visual Studio cache directory
 .vs/
+.vscode/
```

Comment:
We have an existing VSCode template at [`Global\VisualStudioCode.gitignore`](https://github.com/github/gitignore/blob/master/Global/VisualStudioCode.gitignore) that shows what you can be committed and shared.